regexes/oss.yml

```diff
@@ -45,7 +45,7 @@
 # YunOS (Android based)
 ##########
 - regex: '(?:Ali)?YunOS[ /]?(\d+[\.\d]+)?'
-  name: 'YunOS'
+  name: 'YunOs'
```

Comment:
why?  after all , logically O+S is an Operating System.
Almost all sources and news sites indicate the names of `YunOS` or `AliOS`

Reply:
@sgiehl  Probably it is worth correcting the name in the php file ?
https://github.com/matomo-org/device-detector/blob/2468a97d3b3f08c76b38473cb4030eda08c8bef2/Parser/OperatingSystem.php#L141

Reply:
I made the name consistent, as the main repo has `YunOs` in the short name to long name map and `YunOS` in os regexes file.
I agree that `YunOS` seems more correct. The latest commit changes `YunOs` to `YunOS`.